### PR TITLE
Update dockerfile to use 4.17 base image

### PR DESCRIPTION
**Pull request by aneeshkp**
Update dockerfile to use linuxptp 4.2 and synce4l rpm 
This is temporary fix until  base container image is updated to 9.4

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,11 +5,15 @@ COPY . .
...
-FROM registry.ci.openshift.org/ocp/4.16:base-rhel9
+FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
 
+COPY ./extra/linuxptp-4.2-2.el9.x86_64.rpm /tmp/linuxptp-4.2-2.el9.x86_64.rpm
+COPY ./extra/synce4l-1.0.0-1.el9.x86_64.rpm /tmp/synce4l-1.0.0-1.el9.x86_64.rpm
 
-RUN yum -y update && yum -y update glibc && yum --setopt=skip_missing_names_on_install=False -y install linuxptp ethtool hwdata synce4l && yum clean all
+RUN yum -y update && yum -y update glibc && yum --setopt=skip_missing_names_on_install=False -y install ethtool hwdata && yum clean all
 
+RUN rpm -Uvf /tmp/linuxptp-4.2-2.el9.x86_64.rpm
```
Comment (josephdrichard) [inline Dockerfile:15]: Can you add a comment in the dockerfile where the rpm is from?

**Review by nishant-parekh — APPROVED** [review]
/lgtm

**Review by josephdrichard — APPROVED** [review]
/lgtm

**Pull request merged**